BraveRewardsUI/Wallet/RewardsSummaryProtocol.swift

```diff
@@ -49,13 +49,13 @@ extension RewardsSummaryProtocol {
     
     let ledger = state.ledger
     let balance = ledger.balanceReport(for: activityMonth, year: Int32(now.currentYear))
-    
+    let fallback = "-"
     let activities = [
-      Activity(balance.grants, title: Strings.TotalGrantsClaimed, color: BraveUX.adsTintColor),
-      Activity(balance.earningFromAds, title: Strings.EarningFromAds, color: BraveUX.adsTintColor),
-      Activity(balance.autoContribute, title: Strings.AutoContribute, color: BraveUX.autoContributeTintColor),
-      Activity(balance.oneTimeDonation, title: Strings.OneTimeTips, color: BraveUX.tipsTintColor),
-      Activity(balance.recurringDonation, title: Strings.MonthlyTips, color: BraveUX.tipsTintColor)
+      Activity(balance.grants.batValueDisplayString ?? fallback, title: Strings.TotalGrantsClaimed, color: BraveUX.adsTintColor),
```

Comment:
Just update `Activity`'s initializer instead of doing all this & remove the new extension.

RewardsSummaryProtocol.swift:L29:

`guard let value = BATValue(valueString), value.doubleValue != 0.0 else {` → `guard let value = BATValue(probi: valueString), value.doubleValue != 0.0 else {`